src/main/java/Arsdorint/task/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package Arsdorint.task;
+
+public class Deadline extends Task {
+    public static final String typeDeadline = "D";
```

Comment:
Consider renaming your constant variables to all uppercase in order to maintain consistency and adhere to the module's coding standards.